Formula/railway.rb

```diff
@@ -16,11 +16,8 @@ class Railway < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "5f82ce3fe592056a356c7ce02806fd4ed8666b4ea78fa876079bf492ab4e1bf4"
   end
 
-  depends_on "node"
-
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
+    system "./install.sh"
```

Comment:
I think this will try to install pre-built binaries, which is something we don't allow in core repo https://github.com/railwayapp/cli/blob/v2.0.5/install.sh#L27-L34

Ref: https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae